### PR TITLE
feat(config): Add lacework_agent_interface_connection_size variable

### LINKED ISSUE #4: feat(config): Make the "InterfaceConnectionSize" agent parameter configureable
Greetings,

InterfaceConnectionSize is a parameter for the Lacework agent `config.json` file. I need the ability to specify a custom value.

I will work on this feature.

### PR DESCRIPTION
**Pull request by JPLachance**
InterfaceConnectionSize is a new parameter for Lacework agents in the config.json file.

Fixes #4

Signed-off-by: Jean-Philippe Lachance <jplachance@coveo.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ The `main.tf` file will configure a Kubernetes Secret and DaemonSet which will t
 |------|-------------|------|---------|
 | lacework_access_token | The Lacework Access Token for the Datacollector to use | `string` | null |
 | lacework_agent_name | The name for the Lacework agent service within Kubernetes | `string` | "lacework-agent" |
+| lacework_agent_interface_connection_size | Desired value for the InterfaceConnectionSize Lacework agent parameter | `string` | "10000" |
```
Comment (JPLachance) [inline README.md:19]: This new parameter is not documented: https://support.lacework.com/hc/en-us/articles/360024365753-Configure-Agent-Behavior-in-config-json-File

We should write a documentation and place a link to the documentation in the module readme.

File: variables.tf [hunk 2]
```diff
@@ -9,6 +9,12 @@ variable "lacework_agent_name" {
   default     = "lacework-agent"
 }
 
+variable "lacework_agent_interface_connection_size" {
+  type        = string
+  description = "Desired value for the InterfaceConnectionSize Lacework agent parameter"
+  default     = "10000"
```
Comment (JPLachance) [inline variables.tf:15]: Since I was told to use 10000, I placed 10000 as our default value. That being said, what should be the default value?

Comment (afiune) [inline variables.tf:15]: I wonder if these are the default that the agent sets and if not, I wonder what are the implications of setting this by default. I will check with the Agent Team internally.

**Review by afiune — COMMENTED** [review]
Code looks good, just one question about the default value for the new variable.

File: variables.tf (hunk 2 above)
Comment (afiune) [reply variables.tf:15]: LOL!!!! I just asked this same thing! (Amazing minds JP :hi-5:)

**Review by afiune — APPROVED** [review]
tenor-146372473

**Pull request merged**